File: app/src/test/java/com/example/benchmarks/TestSuit.java
package com.example.benchmarks;

import com.example.benchmarks.models.BenchmarkItemUnitTest;
import com.example.benchmarks.models.OperationsCollectionsUnitTest;
import com.example.benchmarks.models.OperationsMapsUnitTest;
import com.example.benchmarks.ui.benchmark.BenchmarksViewModelTest;

import org.junit.runner.RunWith;
import org.junit.runners.Suite;

@RunWith(Suite.class)
@Suite.SuiteClasses({
        BenchmarkItemUnitTest.class,
        BenchmarksViewModelTest.class,
        OperationsCollectionsUnitTest.class,
        OperationsMapsUnitTest.class
})
public class TestSuit {
}

File: app/src/test/java/com/example/benchmarks/TestSuit.kt
package com.example.benchmarks

import org.junit.runner.RunWith
import org.junit.runners.Suite
import org.junit.runners.Suite.SuiteClasses
import com.example.benchmarks.models.benchmark.BenchmarkItemUnitTest
import com.example.benchmarks.ui.benchmark.BenchmarksViewModelTest
import com.example.benchmarks.models.benchmark.OperationsCollectionsUnitTest
import com.example.benchmarks.models.benchmark.OperationsMapsUnitTest

@RunWith(Suite::class)
@SuiteClasses(
    BenchmarkItemUnitTest::class,
    BenchmarksViewModelTest::class,
    OperationsCollectionsUnitTest::class,
    OperationsMapsUnitTest::class
)
class TestSuit 